UnityProject/Assets/Scripts/Tilemaps/Behaviours/Objects/RegisterPlayer.cs

```diff
@@ -291,13 +291,34 @@ public void ServerSlip(bool slipWhileWalking = false)
 			return;
 		}
 
-		ServerStun(StopMovement : false);
+		ServerSlip();
 		AudioSourceParameters audioSourceParameters = new AudioSourceParameters(pitch: Random.Range(0.9f, 1.1f));
 		SoundManager.PlayNetworkedAtPos(CommonSounds.Instance.Slip, WorldPositionServer, audioSourceParameters, sourceObj: gameObject);
 		// Let go of pulled items.
 		playerScript.objectPhysics.StopPulling(false);
 	}
 
+	private void ServerSlip()
+	{
+		ServerCheckStandingChange(true);
+		OnSlipChangeServer.Invoke(IsSlippingServer, true);
+		ServerDropItemsInHand();
+	}
```

Comment:
Wouldn't you need line `350`, `IsSlippingServer = true` here too?

Reply:
slip and stun are the same thing because this code needs to be reworked
we need to know whats the state of `IsSlippingServer` in `ServerSlip()` so if the player was already slipping or slipped the game knows what to do or not do with that infromation.